PageSpeed/GTmetrix/GTMetrixResultViewController.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  GTMetrixResaultViewController.swift
+//  PageSpeed
+//
+//  Created by Admin on 19.02.2020.
```

Comment:
@ilyageekhub why you still use `Admin` as name? It's very confusing and uninformative.

Reply:
fix